identity/client/src/pages/users/List.tsx

```diff
@@ -42,67 +42,87 @@ const List: FC = () => {
 
   const showDetails = ({ username }: User) => navigate(`${username}`);
 
+  const header = (
+    <Stack gap={spacing04}>
+      <PageTitle>
+        <Translate>Users</Translate>
+      </PageTitle>
+      {success && (
+        <PageSubTitle>
+          <Translate>For more information, see documentation on</Translate>{" "}
+          <DocumentationLink path="/concepts/access-control/users">
+            users
+          </DocumentationLink>
+        </PageSubTitle>
+      )}
+    </Stack>
+  );
+
   if (success && !userSearchResults?.items.length) {
     return (
       <Page>
-        <PageTitle>
-          <Translate>Users</Translate>
-        </PageTitle>
-        <C3EmptyState
-          heading={t("You don’t have any users yet")}
-          description={t(
-            "Roles, permissions, and authorizations can be applied to a user.",
-          )}
-          button={{
-            label: t("Create a user"),
-            onClick: addUser,
-          }}
-          link={{
-            href: documentationHref("/concepts/access-control/users", ""),
```

Comment:
This documentation link does not exist in the [latest docs](https://docs.camunda.io/docs/next/components/concepts/access-control/user-groups/) as far as I can see.

Reply:
Docs are definitely due till the minor. Let’s set a reminder in the IAM Core E1 epic that doc links in the UI have to be revised as well.